fuel-core/src/service/config.rs

```diff
@@ -75,3 +85,11 @@ pub enum DbType {
     InMemory,
     RocksDb,
 }
+
+/// A default secret key to use for testing purposes only
+pub fn default_consensus_dev_key() -> SecretKey {
```

Comment:
Should this be under `test-helpers` flag to avoid production use?

Reply:
It may be useful for testing nodes from the CLI when users don't really care about having a real signing key.